deepecho/__init__.py

```diff
@@ -4,6 +4,11 @@
 __email__ = 'info@sdv.dev'
 __version__ = '0.7.1.dev0'
 __path__ = __import__('pkgutil').extend_path(__path__, __name__)
+import os
+
+# Enable fallback so ops not implemented on MPS run on CPU
+# https://github.com/pytorch/pytorch/issues/77764
+os.environ['PYTORCH_ENABLE_MPS_FALLBACK'] = '1'
```

Comment:
This should be removed. This is overriding user's environment variables. I think that if this crashes 'locally' you will get the message about 'enable mps fallback' and you have to do something like:
```bash
export PYTORCH_ENABLE_MPS_FALLBACK=1
```
in your console to fix it.